scli

```diff
@@ -230,14 +259,21 @@ def get_envelope_contact(envelope, signal):
 
 def get_envelope_group_id(envelope):
     try:
-        return envelope['dataMessage']['groupInfo']['groupId']
+        if envelope['dataMessage'] is not None:
+            return envelope['dataMessage']['timestamp']
```

Comment:
I haven't tried the changes but this line should be like this probably, right?
```suggestion
            return envelope['dataMessage']['groupInfo']['groupId']
```

Reply:
Thanks, I've added the `groupInfo` part of it to both normal messages and sync messages